chipy_org/static/less/chipy.less

```
@@ -188,7 +188,7 @@ form {
       font-size: 1.4em;
       }
 
-    .presentor {
+    .presenter {
```

```suggestion
    .presentor {
```
Rolling back because we deleted this file on master and I want to merge quickly.